app/workers/find_and_delete_scheduled_accounts_worker.rb

```diff
@@ -4,6 +4,6 @@ class FindAndDeleteScheduledAccountsWorker
   include Sidekiq::Worker
 
   def perform
-    Account.deleted_time_ago.find_each(&DeleteAccountHierarchyWorker.method(:perform_later))
+    Account.deleted_since.find_each(&DeleteAccountHierarchyWorker.method(:perform_later))
```

Comment:
This was easier to read before, IMO. Perhaps `recently_deleted` instead (or an alias)?

Reply:
Not sure if `recently` is the proper term either 🤔 What this scope does is filtering the accounts  deleted since an specific time, which doesn't need to be recent, and if no time is sent, then it picks the default 🤔 
I've merged but it is not written in stone and can be changed in the next PRs about this if we find a better name 😉